docs/provision.adoc

```diff
@@ -0,0 +1,46 @@
+[[custom-runtime-connector-service]]
+= Custom Runtime Connector
+
+Allows to expose an existing Custom Service to mobile application. Mobile perspective to any existing service could be added
+Custom runtime connector should create a configmap and then you can get the config back by running
+```bash
+oc plugin mobile create client <client-name> <clientType> <bundleID> --namespace <namespace>
+oc plugin mobile get clientconfig <clientID> --namespace=<namespace>
+```
+Where _clientID_(<client-name>-<clientType>) and _namespace_ parameters should be probided by user
+
+There is possibility to represent output in json format
+....
+oc plugin mobile get clientconfig <clientID> --namespace=<namespace> -o json
+....
+
+
+See https://add/link/to/video[this video] which demonstrates how to use CRC
+
+[[provision-custom-runtime-connector-service]]
+== Provision
+Go to the OpenShift web console. Open the *Mobile* Tab > *Services* from the service catalog and select the Custom Runtime Connector service.
+This should open the Custom Runtime Connector service provision wizard. Press the *Next* button and provide the configuration for the service
+
+*The configuration will ask for the following parameters:*
+
+* *Service Name (required)*: Service name to be exposed to mobile application
+* *Service URL (required)*: Service url where service is running
+* *Service JSON Configuration*: Service configuration in JSON format which might be validated
+
+After the service has been provisioned, you are then able to open see secrets and configmaps of service under Resources side menu
+with <service-name> name
+To edit service configuration go to Resources -> Config Maps, click on <service-name> element in table. In next screen you should see
+<service-name> configuration. Press [Actions] button in upper right corner and choose [Edit] or [Edit YAML] whatever your flavor is, that
+allows you to edit directly configuration for service which you can after expose to your mobile application
```

Comment:
`allows you to edit the service configuration directly which you can expose to your mobile application`.